vignettes/flow-driven-spawning-decay.Rmd

```diff
@@ -0,0 +1,525 @@
+---
+title: "Spawning Decay Analysis"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Spawning Decay Analysis}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>", 
+  message = FALSE,
+  warning = FALSE
+)
+
+library(tidyverse)
+library(gt)
+library(gridExtra)
+library(lubridate)
+
+theme_set(theme_bw())
+
+```
+
+```{r setup}
+library(DSMhabitat)
+```
+
+# Objective 
+
+The objective of this analysis is to create flow-decay curves that will be applied to spawning habitat values at runtime in DSM models. The SRH2D model developed as part of the Sediment Study for the Upper Sacramento River serves as the starting point for these curves. Additional "scale-downs" are performed to reduce the magnitude of the curves in order to better match the observed relationship at each watershed where the analysis is applied. Each step in the analysis is detailed below.
```

Comment:
The "Additional "scale-downs" are preformed... sentence is a little confusing to me. You preformed them to best match the watershed where the analysis is applied but don't you also do those scale downs just for sac. I think making it really clear in the objective that this vignette outlines the entire approach for Sac and then maybe a different paragraph of the objective that talks about applying to other watersheds. I know you are still planning on adding that section but I think it would also be good to set it up in the objective.

Reply:
ah yes you are correct, scale-downs are just for the sac, and then final curves are translated to other watersheds, will update!